src/frontend/posts_page/components/SetPublic.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+interface MyProps {
```

Comment:
You can remove this.

Reply:
I guess it is fine to remove every "props" in this file, right?

Reply:
Actually maybe I should keep it. If I remove this, props will become read-only, and this.state.xxx can not be modified.

Reply:
Resolved and thanks the advice from Denise.